packages/extension/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@dailydotdev/react-contexify": "^5.0.2",
     "@dailydotdev/shared": "workspace:*",
+    "@kickass-coderz/react": "^0.0.4",
```

Comment:
Check with @capJavert maybe, but we could also consider not loading the whole suite, but only copy that hook into our own hooks?